js-intermediate.js

```diff
@@ -45,14 +65,24 @@ var originalArray1 = [1, 2, 3, 4, 5, 6, 7]
 var originalArray2 = ["9", "1", "o", "h", "c", "e"]
 // Expected output: ["e", "c", "h", "o", "1", "9"]
 
+const newArray = (array) =>{
+    return array
+}
 
+console.log(newArray(originalArray1.reverse()))
+console.log(newArray(originalArray2.reverse()))
 
```

Comment:
Rather than calling reverse in the console.log put it into your function and return array.reverse()